### PR TITLE
Simple Bugs

### PR DESCRIPTION
**Pull request by manikv12**
Here is my contribution.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,24 +1,16 @@
 ---
 name: Bug report
-about: Create a report to help us improve
```
Comment (seanohue) [inline .github/ISSUE_TEMPLATE/bug_report.md]: This file is meant to be used automatically by GitHub when submitting a bug report issue, no need to commit any changes to it when fixing bugs.

File: ranvier [hunk 2]
```diff
@@ -13,9 +13,9 @@
 
 /* NPM Modules */
 const semver = require('semver');
-const net = require('net');
+//const net = require('net'); Not Being used anywhere in the code
 const commander = require('commander');
-const argv = require('optimist').argv;
+//const argv = require('optimist').argv;   New Node provides argv functuion built in
```
Comment (seanohue) [inline ranvier]: Good catch, though it's preferred to just delete these lines rather than comment them out.

Comment (manikv12) [reply ranvier]: I just wanted to be sure as I am a beginner so i didn't.

**Comment by Kinrany** [comment]
@seanohue @manikv12 would you like me to finish this?

**Pull request closed without merging**